db/db_impl/db_impl.cc

```diff
@@ -2425,6 +2440,11 @@ Status DBImpl::DropColumnFamilyImpl(ColumnFamilyHandle* column_family) {
     assert(cfd->IsDropped());
     ROCKS_LOG_INFO(immutable_db_options_.info_log,
                    "Dropped column family with id %u\n", cfd->GetID());
+    std::unordered_set<std::string> paths;
+    for (auto& cf_path : cfd->ioptions()->cf_paths) {
+      paths.insert(cf_path.path);
+    }
+    s = env_->HintDbPathsRemoved(paths);
```

Comment:
I'm not sure it is a good idea. Different CFs can share the same path. We don't want to remove them unless it is used by another column family.

Reply:
Removed the logic temporarily, will think about how to do it efficiently other than looping through all cf_paths to check conflict.